rdkPlugins/Storage/README.md

```diff
@@ -23,14 +24,42 @@ space "destination".
     }
 }
 ```
+
+### Dynamic Mounts
+Add the following section to your OCI runtime configuration `config.json` file to create dynamic mount. It will mount file from "source" field into container
```

Comment:
Add the information that it will only mount it only if file exist, and if file doesn't exist mount will not created (which is different from normal mounts which are created every time which could result in error) making it dynamic.
Also you could add information that for now file should have ownership set prior for the container to use the file inside)

Reply:
Done